src/main/java/net/pms/network/HTTPResource.java

```diff
@@ -44,6 +44,7 @@
 	public static final String UNKNOWN_AUDIO_TYPEMIME = "audio/mpeg";
 	public static final String AUDIO_TRANSCODE = "audio/transcode";
 	public static final String VIDEO_TRANSCODE = "video/transcode";
+	public static final String AUDIO_AAC_TYPEMIME = "audio/mp4";
```

Comment:
Really ?

> 8.9      MPEG-4 profiling guidelines
> 8.9.1       General
> The DLNA-defined profiles of MPEG-4 are based on MPEG-2/4 AAC, MPEG-4 High Efficiency AAC, and MPEG-4 High Efficiency AAC v2. There are guidelines for both stereo and multichannel audio and several supported file formats. Many of the audio profiles incorporating AAC have a hierarchical relationship to each other. A DMP capable of rendering any of the multichannel AAC profiles is also capable  of  rendering  the  corresponding  profiles  with  fewer  channels.  In  addition,  some  profiles differ only in whether a bit rate restriction is imposed. A hierarchical relationship exists here, too, where a DMP capable of rendering a non-restricted bit rate profile is also capable of rendering the corresponding   profile   with   a   bit   rate   restriction.   Furthermore,   the   "HE-AAC"   profiles   and "HE-AACv2" profiles are each a superset of a corresponding "AAC" profile because they all use the AAC-LC audio object. The "HE-AACv2" profiles are also a superset of a corresponding "HE-AAC" profile.

> 8.9.55    AAC audio format: MIME type definition: MP4, 3GPP
> 8.9.55.1
> [PROFILES]
> AAC_ISO
> AAC_ISO_320
> AAC_ISO_192
> AAC_MULT5_ISO
> HEAAC_L2_ISO
> HEAAC_L2_ISO_320
> HEAAC_L2_ISO_128
> HEAAC_MULT5_ISO
> 8.9.55.2
> [GUIDELINE]    MIME type "audio/mp4"or "audio/3gpp" shall be used for these Media Format Profiles, 
> depending on the audio interchange format used.
> [ATTRIBUTES]
> M     R      n/a n/a n/a ISO/IEC
> 14496-12
> 3GPP TS 
> 26.244
> QRUUV
> 8.9.56    AAC audio format: MIME type definition: MP4, 3GPP, ADTS
> 8.9.56.1
> [PROFILES]
> AAC_MPS
> MPEG2_AAC_MPS
> HEAAC_L4
> HEAAC_MPS
> HEAACv2_L2
> HEAACv2_L2_320
> HEAACv2_L2_128
> HEAACv2_L4
> 8.9.56.2
> [GUIDELINE]    MIME  type  "audio/mp4",  "audio/3gpp",  or  "audio/vnd.dlna.adts"  shall  be  used  for 
> these Media Format Profiles, depending on the Audio Interchange Format used.